lib/pure/parseopt.nim

```diff
@@ -75,27 +75,27 @@
 ## The `getopt iterator<#getopt.i,OptParser>`_, which is provided for
 ## convenience, can be used to iterate through all command line options as well.
 ##
-## ``shortNoVal`` and ``longNoVal``
+## `shortNoVal` and `longNoVal`
 ## ================================
```

Comment:
Nit: You may want to adjust the amount of `=`s here.

Reply:
thanks